### ISSUE TITLE
String improvement "Failed to encrypt message\nAccount has no public key"

### ISSUE THREAD

**Issue by fiazk** [NewEconomyMovement/NEMiOSApp#184]
When the user tries to send an encrypted message for the 1st time and gets this it's not clear enough what's the problem and how it can be solved.

Perhaps it could be something like this "Failed to encrypt message\nAccount needs to become public first by sending a transaction"

**Comment by jabo38** [comment]
yes, generally through the whole app the error messages aren't clear.

**Comment by fiazk** [comment]
@jabo38 @fyahfox could someone please check if this was fixed or send me SS since i dont have any iOS device. thx
